### PR TITLE
Package imutils

### LINKED ISSUE #8562: Packaging imutils
From https://github.com/jrosebr1/imutils resp. https://pypi.org/project/imutils/

### PR DESCRIPTION
**Pull request by h-vetinari**
Checklist
closes #8562

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

Currently listed only myself as maintainer, but hoping one of the people [tagged below](https://github.com/conda-forge/staged-recipes/pull/8572#issuecomment-502962396) will jump in as well.

### CODE REVIEW

**Comment by h-vetinari** [comment]
Optimistically tagging the package owner @jrosebr1 for possible maintainership, as well as some people who have [already](https://anaconda.org/search?q=imutils) built a conda package for imutils: @pjamesjoyce @mlgill @gilbertfrancois @chriskan0305 @akode @thecampfire.

Would be happy to share the responsibility. :)

**Comment by h-vetinari** [comment]
Seems the simple recipe based on `conda skeleton pypi` doesn't work everywhere (it did build locally for me). Will have to leave the debugging for later.

**Comment by mlgill** [comment]
Please untag me from this discussion. Thank you.

**Comment by h-vetinari** [comment]
> @mlgill: Please untag me from this discussion. Thank you.

I can't do that for you, you'll have to click "unsubscribe" under "Notifications" in the top right-hand corner of the thread.

**Comment by jrosebr1** [comment]
Sorry @h-vetinari, but I cannot help in that regard. Between authoring the free PyImageSearch posts, helping others learn CV and DL, and of course, maintaining `imutils` itself, I just don't have the time for anything else.

**Comment by h-vetinari** [comment]
Currently running into the issue that libGL is not found on linux, which [seems to be intentional](https://github.com/conda-forge/pygridgen-feedstock/issues/10#issuecomment-365983589). I'm trying to glean some tricks from the `meta.yaml` of the feedstocks for opencv, qt and matplotlib. Let's see if this get's me anywhere.

File: recipes/imutils/yum_requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+mesa-libGL
```
Comment (scopatz) [inline recipes/imutils/yum_requirements.txt]: You might need to use `mesa-libGL-devel` if you need the headers

Comment (scopatz) [reply recipes/imutils/yum_requirements.txt]: But it looks like you don't use the headers, per se

Comment (h-vetinari) [reply recipes/imutils/yum_requirements.txt]: Opencv [doesn't](https://github.com/conda-forge/opencv-feedstock/blob/master/recipe/yum_requirements.txt) use them either. Frankly, all I tried to do was to get a passing build. 😅

Comment (scopatz) [reply recipes/imutils/yum_requirements.txt]: Which it seems like you have done!

Comment (scopatz) [reply recipes/imutils/yum_requirements.txt]: Let us know when this is ready for review

**Comment by h-vetinari** [comment]
> @scopatz: Let us know when this is ready for review

From my POV, this is ready! The PR template already mentioned that I cannot (yet) tag @conda-forge/help-python, so thanks for stopping by. :)

I would of course gladly accept more recipe maintainers, but as no-one has spoken up in about 36h, I think I'll just bite the bullet.

**Comment by scopatz** [comment]
Thanks @h-vetinari!

**Pull request merged**